File: java/src/test/java/com/libutil/test/file/EncodeBase64FileToFileTest.java
package com.libutil.test.file;

import java.io.IOException;

import com.libutil.FileUtil;
import com.libutil.test.Log;

public class EncodeBase64FileToFileTest {

  public static void main(String args[]) {
    try {
      test();
    } catch (IOException ioe) {
      Log.e(ioe);
    }
  }

  private static void test() throws IOException {
    FileUtil.encodeBase64FileToFile("C:/test/image.jpg", "C:/tmp/b64_newline.txt");
    FileUtil.encodeBase64FileToFile("C:/test/image.jpg", "C:/tmp/b64.txt", 0);
    FileUtil.encodeBase64FileToFile("C:/test/NotFound.txt", "C:/tmp/b64_not_found.txt");
    Log.d("OK");
  }

}

File: python/test/base64/encode_base64_file_to_file_test.py
import os
import sys

sys.path.append(os.path.join(os.path.dirname(__file__), '../..'))
import util

def main():
    util.encode_base64_file_to_file('C:/test/image.jpg', 'C:/tmp/b64_newline.txt')
    util.encode_base64_file_to_file('C:/test/image.jpg', 'C:/tmp/b64_1.txt', 0)
    print('OK')

main()
